x-pack/docs/en/security/authentication/jwt-realm.asciidoc

```diff
@@ -43,18 +45,17 @@ h| Information     | ID Token   | Access Token
 In addition, {es} also validates `nbf` and `auth_time` claims for ID Tokens if these claims are present.
 But these claims are ignored for access tokens.
 
-A single JWT realm can only work with a single token type. The default token
-type is `id_token`. To handle both token types, you must configure at least
-two JWT realms.
+Overall, the access token type has more relaxed validation rules and is suitable for more generic JWTs,
+including self-signed ones.
 
 [[jwt-realm-oidc]]
-==== JWT from OIDC workflows
+==== ID Tokens from OIDC workflows
 JWT authentication in {es} is derived from OIDC user workflows, where different
-tokens can be issued by an OIDC Provider (OP), including ID Tokens and access
-tokens. ID Tokens are well-defined JWT and should be always compatible with
-what JWT realm supports. Access tokens can be arbitrary in theory. For them to
-be usable with JWT realm, they must at least use the JWT format and satisfy
-relevant requirements in the above table.
+tokens can be issued by an OIDC Provider (OP), including ID Tokens.
+ID Tokens are well-defined JWT and should be always compatible with
+a JWT realm of the `id_token` token type. ID Tokens usually represent end-users
+which can be of a large number. Therefore, a JWT realm of `id_token` token type
+does _not_ enforce the `allowed_subjects` validation.
```

Comment:
```suggestion
a JWT realm of the `id_token` token type.
```

I don't that this is needed to call out something not required. If you want to keep it I would suggest using the term "not mandated" to better match the text below.

Reply:
I prefer to keep it to highlight the difference between the `id_token` and `access_token` configuration. I updated it to say "does not mandate ...".